dynamiq/nodes/agents/orchestrators/adaptive.py

```diff
@@ -47,13 +47,15 @@ class AdaptiveOrchestrator(Orchestrator):
         agents (List[BaseAgent]): List of specialized agents available for task execution.
         objective (Optional[str]): The main objective of the orchestration.
         max_loops (Optional[int]): Maximum number of actions.
+        enable_reflection (Optional[bool]): Enable reflection mode
     """
 
     name: str | None = "AdaptiveOrchestrator"
     group: NodeGroup = NodeGroup.AGENTS
     manager: AdaptiveAgentManager
     agents: list[Agent] = []
     max_loops: int = 15
+    enable_reflection: bool = False
```

Comment:
Let's use `reflection_enabled` similar to our  existing style like `Config.enabled` and `files_allowed`